src/main/java/com/mycompany/myapp/service/MailService.java

```diff
@@ -82,7 +88,20 @@ public void sendEmailFromTemplate(User user, String templateName, String titleKe
         String content = templateEngine.process(templateName, context);
         String subject = messageSource.getMessage(titleKey, null, locale);
         sendEmail(user.getEmail(), subject, content, false, true);
+    }
 
+    @Async
+    public void sendToolsRequestEmail(User user, String poolId, String[] tools){
+        log.debug("Sending request for additional tools for pool '{}'", poolId);
+        Locale locale = Locale.forLanguageTag(user.getLangKey());
+        Context context = new Context(locale);
+        context.setVariable(USER, user);
+        context.setVariable(BASE_URL, jHipsterProperties.getMail().getBaseUrl());
+        context.setVariable(POOL_ID, poolId);
+        context.setVariable(TOOLS, Arrays.asList(tools));
+        String content = templateEngine.process("mail/requestToolsEmail", context);
+        String subject = messageSource.getMessage("email.request.title", null, locale);
+        sendEmail("snorlax.io2019@gmail.com", subject, content, false, true); //TODO change emails to admins
```

Comment:
That's not a TODO. We have application configs for all Spring profiles (dev, prod and test). Add it there